examples/context/motivation-problem.js

```diff
@@ -1,15 +1,15 @@
 class App extends React.Component {
   render() {
-    return <Toolbar theme="dark" />;
+    return <Toolbar theme="foncé" />;
```

Comment:
```suggestion
    return <Toolbar theme="dark" />;
```

On ne traduit pas les identifiants, etc. Seulement les textes destinés à l'affichage.